runtest.py

```diff
@@ -237,6 +227,8 @@ def loop_test(self, test_case_module_name, test_name, test_env, current_run=0, c
         while current_run < self.exec_config['max-run']:
             self.logger.info("The counter is %d and the retry counter is %d" % (current_run, current_retry))
             try:
+                if self.online_config['enable'] and os.path.exists(self.default_result_fp):
+                    os.remove(self.default_result_fp)
```

Comment:
It will helpful if we add comments to describe we will remove `result.json` every time on Online mode.